data.sql

```diff
@@ -55,3 +54,149 @@ WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
 UPDATE animals
 SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
 WHERE name = 'Angemon'  OR name = 'Boarmon';
+
+/* Vets*/
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'William Tatcher', 45, '23/04/2000');
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'Maisy Smith', 26, '17/01/2019');
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'Stephaine Mendez', 64, '04/05/1981');
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES(
+'Jack Harkness', 38, '08/06/2008');
+
+/* Specialisations */
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Pokemon'),
+  (SELECT id FROM vets WHERE name = 'William Tatcher')
+);
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Digimon'),
+  (SELECT id FROM vets WHERE name = 'Stephaine Mendez')
+);
+
+INSERT INTO specializations (species_id, vet_id) VALUES( 
+  (SELECT id FROM species WHERE name = 'Digimon'),
+  (SELECT id FROM vets WHERE name = 'Jack Harkness')
+);
+
+
+/*Visits*/
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES(
+(SELECT id FROM animals WHERE name = 'Agumon'),
+(SELECT id FROM vets WHERE name = 'Stephaine Mendez'),
+'22/07/2020');
+
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES(
+(SELECT id FROM animals WHERE name = 'Gabumon'),
+(SELECT id FROM vets WHERE name = 'Jack Harkness'),
+'02/02/2021');
+
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES(
+(SELECT id FROM animals WHERE name = 'Pikachu'),
+(SELECT id FROM vets WHERE name = 'Maisy Smith'),
+'05/01/2020');
+
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES(
+(SELECT id FROM animals WHERE name = 'Pikachu'),
+(SELECT id FROM vets WHERE name = 'Maisy Smith'),
```

Comment:
- [ ] You missed adding the following data to `visits`, Please make sure to add that.
  - Agumon visited William Tatcher on May 24th, 2020.